Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.10-alpine
+LABEL maintainer="leon.kushnir15@gmail.com"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
+RUN mkdir -p /files/media

```

Comment:
Consider creating the `/files/media` directory before copying the application files. This ensures that the directory is set up correctly before any application code is added.